test/file_pattern_matcher_test.py

```diff
@@ -272,11 +272,6 @@ def test_match():
             assert FilePatternMatcher(pattern)._matches(text) is expected
 
 
-def test_absolute_path_error():
```

Comment:
should we keep this test but assert that absolute paths work as we expect them to - as a regression test?

Reply:
Hm yeah I added handling of leading / trailing slashes to the basic `test_match` test in `file_pattern_matching_test`